pages/checklist/index.tsx

```diff
@@ -0,0 +1,676 @@
+import Head from 'next/head'
```

Comment:
General thoughts: can we make this smaller? it's really big for a page and is loading slowly as a result. can you write the text in the constants folder and map that?

Also, can we do this at 90% width and center by default?